internal/input/assets_history_archive.go

```diff
@@ -0,0 +1,51 @@
+package input
+
+import (
+	"context"
+
+	"github.com/stellar/stellar-etl/internal/transform"
+	"github.com/stellar/stellar-etl/internal/utils"
+
+	"github.com/stellar/go/xdr"
+)
+
+// GetPaymentOperations returns a slice of payment operations that can include new assets from the ledgers in the provided range (inclusive on both ends)
+func GetPaymentOperationsHistoryArchive(start, end uint32, limit int64, isTest bool, isFuture bool) ([]AssetTransformInput, error) {
```

Comment:
even though the backend is different, might be nice to maintain parity between `GetPaymentOperations` and `GetPaymentOperationsHistoryArchive`. ie, maybe keep the input parameters for the methods consistent.